types/net/net.d.ts

```diff
@@ -0,0 +1,70 @@
+declare module 'stripe' {
+  namespace Stripe {
+    export namespace HttpClient {
+      /**
+       * Abstract representation of an HTTP response. This is an experimental
+       * interface and is not yet stable.
+       */
+      interface Response {
+        /** The numeric HTTP status code for the response. */
+        getStatusCode(): number;
+
+        /** The response headers. */
+        getHeaders(): {[key: string]: string};
+
+        /** This returns the underlying raw response object for the client. */
+        getRawResponse(): any; //eslint-disable-line @typescript-eslint/no-explicit-any
+
+        /**
+         * This returns the content as a stream. This is implementation-specific
+         * and so we cannot use a given type as different platforms have
+         * different stream primitives.The expectation is that content
+         * will not have been buffered into memory at this point by the client.
+         *
+         * The streamCompleteCallback should be invoked by the response
+         * implementation when the stream has been consumed.
+         */
+        toStream(streamCompleteCallback: () => void): any; //eslint-disable-line @typescript-eslint/no-explicit-any
+
+        /**
+         * Converts the response content into a JSON object, failing if JSON
+         * couldn't be parsed.
+         */
+        toJSON(): Promise<object>;
```

Comment:
Do we have a union type that's, like, all possible stripe resource types?

Reply:
I don't think so unfortunately. @richardm-stripe to confirm?

Reply:
We don't but could easily have one if we wanted.

Though `object` feels better to me here, since `HTTPClient` should be mostly stripe-domain-agnostic.